OZON.ru.js

```diff
@@ -406,24 +131,18 @@ var testCases = [
 						"creatorType": "author"
 					}
 				],
-				"notes": [],
-				"tags": [],
-				"seeAlso": [],
-				"attachments": [
-					{
-						"title": "OZON.ru Link",
-						"mimeType": "text/html",
-						"snapshot": false
-					}
-				],
-				"title": "Язык программирования C++",
-				"language": "ru",
-				"publisher": "Бином, Невский Диалект",
-				"ISBN": "5-7940-0031-7, 5-7989-0127-0, 0-201-88954-4",
-				"date": "1999-1-1",
-				"numPages": "991",
+				"date": "1999",
+				"ISBN": "9785794000313",
 				"abstractNote": "Книга Бьерна Страуструпа является каноническим изложением возможностей С++, написанным автором этого популярнейшего языка программирования. Помимо подробного описания самого языка, на страницах книги вы найдете множество доказавших свою эффективность подходов к решению разнообразных задач программирования и проектирования. Многочисленные примеры демонстрируют как хороший стиль программирования на С - совместимом ядре С++, так и современный объектно - ориентированный подход к созданию программных продуктов. Данное третье издание бестселлера было существенно переработано автором. С одной стороны, результатом этой переработки стала большая доступность книги для новичков. С другой стороны, текст обогатился сведениями и техниками программирования, которые могут оказаться полезными даже для многоопытных специалистов по С++. Разумеется, не обойдены вниманием и нововведения языка: стандартная библиотека шаблонов (STL), пространства имен (namespaces), механизм идентификации типов во время выполнения (RTTI), явныеприведения типов (cast - операторы) и другие. Книга адресована программистам, использующим в своей повседневной работе С++. Она также будет полезна преподавателям, студентам и всем, кто хочет ознакомиться с описанием языка `из первых рук`.",
-				"libraryCatalog": "OZON.ru"
+				"language": "Русский",
+				"libraryCatalog": "OZON.ru",
+				"numPages": "991",
+				"publisher": "Бином, Невский Диалект",
+				"url": "http://www.ozon.ru/context/detail/id/87889/",
```

Comment:
Actually, the URL should point to the website with an available full-text resource (hosted online). We do not include URLs to catalog entries.